### ISSUE TITLE
API client - token blacklist 

### ISSUE THREAD

**Issue by sambles** [OasisLMF/OasisLMF#441]
* Issue 1:  API token timeout not set based on environment vars 
* Issue 2: Client fails to handled expired refresh token

Error Trace:
```
Traceback (most recent call last):
 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\session.py", line 144, in get

raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: http://oak-land:8000/V1/analyses/125/
During handling of the above exception, another exception occurred:

 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\client.py", line 471, in run_analysis
   r = self.analyses.get(analysis_id)
 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\client.py", line 45, in get
   return self.session.get('{}{}/'.format(self.url_endpoint, ID))
 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\session.py", line 147, in get
   if self.__recoverable(e, url, 'GET', counter):
 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\session.py", line 103, in __recoverable
   self._refresh_token()
 File "C:\ProgramData\Anaconda3\envs\OasisVM\lib\site-packages\oasislmf\api\session.py", line 67, in _refresh_token
   raise OasisException(err_msg)
oasislmf.utils.exceptions.OasisException: Token refresh error: {"detail":"Token is blacklisted","code":"token_not_valid"}`
```

Token Compose setup:
```
     - OASIS_TOKEN_ACCESS_LIFETIME = days=10
     - OASIS_TOKEN_REFRESH_LIFETIME = days=10
```

**Comment by sambles** [comment]
1) whitespace causes the default value to be set, it should look like
`- OASIS_TOKEN_REFRESH_LIFETIME=days=10`

2) when `TOKEN_REFRESH_ROTATE = True` is set (on by default) the client is not updating the refresh token once its re-issued

**Comment by sambles** [comment]
fix: 61b9b44ce3988f7ddf95d946e7e2665e59f8a10f
